src/Message/AbstractOrderRequest.php

```diff
@@ -41,6 +42,14 @@ public function getShippingAddress()
     }
 
     /**
+     * @return Options
+     */
+    public function getOptions()
```

Comment:
Wouldn't `getWidgetOptions` be a more expressive name? Same goes for the VO and setter.